### PR TITLE
JBPM-7128 - Extend test-coverage of toolbar commands

### PR DESCRIPTION
**Pull request by domhanak**
Add unit test for each toolbar command and
verify instance of these commands are correctly created.

@hasys @romartin Could you please take a look? Adds unit tests. Integration test for toolbar are in progress

### CODE REVIEW

**Comment by romartin** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/ClearStatesToolbarCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/ClearStatesToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/ClearToolbarCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/ClearToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/CopyToolbarCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/CopyToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/CutToolbarCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/CutToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/DeleteSelectionToolbarCommandTest.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/DeleteSelectionToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/PasteToolbarCommandTest.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/PasteToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/RedoToolbarCommandTest.java [hunk 7]
```diff
@@ -0,0 +1,46 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/RedoToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/SwitchGridToolbarCommandTest.java [hunk 8]
```diff
@@ -0,0 +1,50 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/SwitchGridToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/UndoToolbarCommandTest.java [hunk 9]
```diff
@@ -0,0 +1,44 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/UndoToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/ValidateToolbarCommandTest.java [hunk 10]
```diff
@@ -0,0 +1,44 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/ValidateToolbarCommandTest.java]: License please.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/VisitGraphToolbarCommandTest.java [hunk 11]
```diff
@@ -0,0 +1,44 @@
+package org.kie.workbench.common.stunner.client.widgets.toolbar.command;
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/VisitGraphToolbarCommandTest.java]: License please.

**Comment by romartin** [comment]
oh well seen @manstis - missing all licenses!

**Comment by domhanak** [comment]
@romartin Yup somehow I suddenly had no definitions of it in IDE - fixed.

**Comment by domhanak** [comment]
Jenkins retest this

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/ClearToolbarCommandTest.java [hunk 12]
```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/toolbar/command/ClearToolbarCommandTest.java]: 2017?? You can use [this template](https://github.com/romartin/droolsjbpm-build-bootstrap/blob/master/ide-configuration/LICENSE-ASL-2.0-HEADER.txt) for the license, it'll use the right year.

**Comment by domhanak** [comment]
@romartin updated sorry for that

**Comment by domhanak** [comment]
jenkins retest this

**Comment by domhanak** [comment]
jenkins execute full downstream build

**Comment by domhanak** [comment]
jenkins retest this

**Pull request merged**